install_script.sh.template

```diff
@@ -577,39 +553,6 @@ function filter_packages_installed_by_installer() {
     eval "$pkg_array_name=(\"\${not_installed_packages[@]}\")"
 }
 
-# Install the Datadog installer package
-function install_installer() {
-    local sudo_cmd="$1"
-    local os="$2"
-    local distribution="$3"
-    local datadog_installer="$4"
-    local remote_updates="$5"
-    local apm_instrumentation_enabled="$6"
-    local pkg_array_name="$7"
-
-    local trace_id
-    local start_time
-    local exit_status
-
-    # The installer is currently only being rolled out to APM instrumentation users
-    if [ -z "$apm_instrumentation_enabled" ] && [ -z "$datadog_installer" ] && [ -z "$remote_updates" ]; then
```

Comment:
I see. This line also allowed us to install the installer without installing APM by setting `DD_INSTALLER=true`